docs/src/man/getting_started.md

```diff
@@ -128,14 +128,22 @@ dt = DataTable(A = 1:4, B = randn(4))
 colwise(c->cumsum(dropnull(c)), dt)
 ```
 
+
+## IO
```

Comment:
"IO" is a pretty technical term for newcomers. "Importing Data"?

Also, one line break is enough.